flowzone.yml

```diff
@@ -2998,6 +3006,9 @@ jobs:
       matrix:
         value: ${{ fromJSON(needs.is_custom.outputs.custom_test_matrix) }}
         os: ${{ fromJSON(inputs.custom_runs_on || format('[{0}]', inputs.runs_on)) }}
+        include: ${{ fromJSON(inputs.custom_matrices_includes) }}
+
+    environment: ${{ matrix.environment || null }}
```

Comment:
I think environment(s) should be fully supported by Flowzone at the top level and not require custom matrices, even though this looks like it would solve the immediate problem at hand.

Can we brainstorm about a more generic approach next week?